### ISSUE TITLE
Nostlan hangs on startup with "cui is not defined"

### ISSUE THREAD

**Issue by Hydrox6** [quinton-ashley/nostlan#11]
The application opens, loads the Power, Reset, and Open buttons, along with the Nostlan logo, but all do nothing. In the console, the error

```
ReferenceError: cui is not defined
    at module.exports (index.js:574)
```

happens in `index.js`

Running on Linux Mint, using the AppImage for v1.3.2

**Comment by quinton-ashley** [comment]
thanks for reporting this, I'll look into it!

**Comment by quinton-ashley** [comment]
should be fixed in v1.4.x 🎊
